.github/workflows/maven-ci.yml

```diff
@@ -43,6 +41,12 @@ jobs:
       uses: actions/setup-java@v1
       with:
         java-version: 11
+    - uses: actions/cache@v1
```

Comment:
It seems there's cache@v2 available https://github.com/actions/cache Probably we should use the latest? Also see https://docs.github.com/en/actions/configuring-and-managing-workflows/caching-dependencies-to-speed-up-workflows

Reply:
Yes I made the changes.
Please review the changes @iluwatar 

Thanks.